test/Common/DurableTaskLifeCycleNotificationTest.cs

```diff
@@ -1,1340 +1,1340 @@
-            {
-                await host.StartAsync();
-
-                var client = await host.StartOrchestratorAsync(
-                    functionName,
-                    0,
-                    this.output,
-                    createdInstanceId);
-
-                await client.WaitForStartupAsync(TimeSpan.FromSeconds(30), this.output);
-                await client.TerminateAsync("sayōnara");
-
-                var status = await client.WaitForCompletionAsync(TimeSpan.FromSeconds(30), this.output);
-
-                Assert.Equal(OrchestrationRuntimeStatus.Terminated, status?.RuntimeStatus);
-                Assert.Equal("sayōnara", status?.Output);
-
-                // There should be one validator for each Event Grid request.
-                // Each validator is a delegate with several Assert statements.
-                Assert.NotEmpty(eventGridRequestValidators);
-                foreach (Action validator in eventGridRequestValidators)
-                {
-                    validator.Invoke();
-                }
-
-                // TODO: There should be two calls, but the termination notification is not being fired.
-                //       https://github.com/Azure/azure-functions-durable-extension/issues/286
-                Assert.Equal(1, callCount);
-                await host.StopAsync();
-            }
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using Moq.Protected;
+using Newtonsoft.Json.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests
+{
+    public class DurableTaskLifeCycleNotificationTest
+    {
+        private readonly ITestOutputHelper output;
+        private readonly TestLoggerProvider loggerProvider;
+        private readonly bool useTestLogger = true;
+
+        public DurableTaskLifeCycleNotificationTest(ITestOutputHelper output)
+        {
+            this.output = output;
+            this.loggerProvider = new TestLoggerProvider(output);
         }
-
-        [Theory]
-        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
-        [InlineData(true)]
-        [InlineData(false)]
-        public async Task OrchestrationStartedOptOutOfEvent(bool extendedSessionsEnabled)
-        {
-            var testName = nameof(this.OrchestrationStartAndCompleted);
-            var functionName = nameof(TestOrchestrations.SayHelloInline);
-            var eventGridKeyValue = "testEventGridKey";
+
+        [Theory]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory + "_BVT")]

```

Comment:
Out of curiosity, what is the difference between `PlatformSpecificHelpers.TestCategory` and `PlatformSpecificHelpers.TestCategory + "_BVT"`.

Reply:
It has to do with how we set up our CI to run automated tests. Here is a screenshot:

![image](https://user-images.githubusercontent.com/2704139/55022177-48c52580-4fb7-11e9-9b6d-1765f3518146.png)

Basically, anything that has `_BVT` at the end of it will be run in both the V1 and V2 test projects. Anything without this will only run in the V2 test projects. We had to do this because the V1 tests were running extremely slowly over time, and becoming unreliable. So basically, we only run the most important tests against Functions V1.